### PR TITLE
Add explicit model mapping

### PR DESCRIPTION
**Pull request by pbylicki**

### CODE REVIEW

File: rfhub2/db/model/keyword.py [hunk 1]
```diff
@@ -24,3 +25,36 @@ def __repr__(self):  # pragma: no cover
     @classmethod
     def default_ordering(cls) -> List[OrderingItem]:
         return [OrderingItem("name")]
+
+    @staticmethod
+    def create(data: model.KeywordCreate) -> "Keyword":
+        return Keyword(
+            name=data.name,
+            doc=data.doc,
+            args=data.args,
```
Comment (pbylicki) [inline rfhub2/db/model/keyword.py:34]: @MaciejWiczk you could add here conversion of `tags: List[str]` from `data` to `tags: str` in DB model by using json serialization

File: rfhub2/db/model/keyword.py [hunk 2]
```diff
@@ -24,3 +25,36 @@ def __repr__(self):  # pragma: no cover
...
+            collection_id=data.collection_id,
+        )
+
+    def to_model(self) -> model.Keyword:
+        collection = self.collection
+        return model.Keyword(
+            id=self.id,
+            name=self.name,
+            doc=self.doc,
+            args=self.args,
```
Comment (pbylicki) [inline rfhub2/db/model/keyword.py:44]: @MaciejWiczk you could add here conversion of `tags: str` in DB model to `tags: List[str]` in `mode.Keyword` by using json deserialization

File: rfhub2/db/model/keyword.py (hunk 1 above)
Comment (MaciejWiczk) [reply rfhub2/db/model/keyword.py:34]: What do You mean by json serialization? json.loads and then some list comprehension or something totally different? Or using json encoder like [here](https://fastapi.tiangolo.com/tutorial/encoder/) just on tags object?

**Comment by MaciejWiczk** [comment]
Before merging this one we should update version.py. If those changes are not be backward compatible.

**Comment by pbylicki** [comment]
> Before merging this one we should update version.py. If those changes are not be backward compatible.

Could you explain? These changes fix wrong behavior of some of the endpoints that used to work with old versions of fastapi, but don't work with latest ones. There is no change in the API schema.
My goal was to merge it so you can use it in your MR with tags so that they can be released together.

**Review by MaciejWiczk — APPROVED** [review]
> > Before merging this one we should update version.py. If those changes are not be backward compatible.
> 
> Could you explain? These changes fix wrong behavior of some of the endpoints that used to work with old versions of fastapi, but don't work with latest ones. There is no change in the API schema.
> My goal was to merge it so you can use it in your MR with tags so that they can be released together.

oh, if this is perfectly backward compatible, than sure. I was afraid that client from newest version would mismatch with apprunning on version 0.14 or lower.

File: rfhub2/db/model/keyword.py (hunk 1 above)
Comment (pbylicki) [reply rfhub2/db/model/keyword.py:34]: json.loads/json.dumps will be enough for now

**Comment by pbylicki** [comment]
@MaciejWiczk One thing, that came to my mind is that you could want to have this released before tags, so users can use this changes removing fastapi version limitation without using the new tags functionality. But I don't think that's really an issue

**Review by MaciejWiczk — COMMENTED** [review]
Yes, I agree that this can be released as is, before tags. But without bumping version this will not go to pypi.

**Comment by pbylicki** [comment]
@MaciejWiczk Sure, just wanted to get your thoughts. Bumped to 0.15

**Pull request merged**